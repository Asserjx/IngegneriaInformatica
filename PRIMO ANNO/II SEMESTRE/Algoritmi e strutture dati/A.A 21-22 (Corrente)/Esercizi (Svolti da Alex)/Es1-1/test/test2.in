10
-3
-2
-6
2
-10
4
-6
-1
1
1
