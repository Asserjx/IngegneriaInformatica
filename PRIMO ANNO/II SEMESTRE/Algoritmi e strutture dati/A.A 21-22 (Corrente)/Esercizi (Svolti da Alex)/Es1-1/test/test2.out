-20
17280
no
